introduction.md

```diff
@@ -0,0 +1 @@
+My name is Sam and I go by irritablekitten on Github. I am honored to join this dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.